registry/handlers/hooks.go

```diff
@@ -0,0 +1,28 @@
+package handlers
+
+import (
+	"github.com/Sirupsen/logrus"
+)
+
+// LogHook is for hooking Panic in web application
+type LogHook struct {
+	LevelsParam []string
+	MailParam   Mail
```

Comment:
Could hook embed the configuration Options struct?  It contains all the information you need for MailSending and is auto populated from the configuration.